packages/www/components/Dashboard/AssetsTable/CreateAssetDialog.tsx

```diff
@@ -184,6 +186,8 @@ const CreateAssetDialog = ({
               variant="gray"
               css={{ mt: "$1", fontSize: "$2", mb: "$4" }}>
               Select up to 20 files at a time
+              <br />
+              .mp4 files supported
```

Comment:
Should we have an "only"?
```suggestion
              only .mp4 files supported
```